%YAML:1.0

camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: f
   data: [926.119019, 0, 623.334473, 0, 925.037476, 365.230743, 0, 0, 1 ]
distortion_coefficients: !!opencv-matrix
   rows: 5
   cols: 1
   dt: f
   data: [ 0., 0., 0. , 0., 0.]


